alerta/auth/oidc.py

```diff
@@ -73,6 +73,10 @@ def openid():
     r = requests.post(token_endpoint, data)
     token = r.json()
 
+    if 'error' in token:
+        error_text = token.get('error_description') or token['error']
+        raise ApiError(error_text)
+
```

Comment:
This captures any errors returned by the identity provider and makes them visible in the web UI.